README.md

```diff
@@ -56,7 +55,7 @@ REQUIRED SECTIONS:
 
```

Comment:
![image](https://github.com/otmaneechchafyky/bookstore/assets/93039359/648563be-c37b-4526-9a48-65ed2218629d)

**[OPTIONAL]**

- [ ] It would be nice if you remove one of the **back to top** buttons from the `Key Features` section to make your readme descriptive and professional :+1: